Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-ruby '2.6.6'
```

Comment:
@Reddshift , might be best to get this updated to the latest 2.x version. 
Also we can just remove including the patch version in the Gemfile so you can probably have this be `ruby '2.7'`

Reply:
Great point! - I can't even remember why I changed it 😆